### PR TITLE
Bulk title missing fixes

### PR DESCRIPTION
**Pull request by jmirabal80**

### CODE REVIEW

File: Skype/UCWA/invitation_ref.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+title: invitation
```
Comment (LizCasey1) [inline Skype/UCWA/invitation_ref.md:2]: Please make this a capital letter, thanks.

File: Skype/UCWA/myContacts_ref.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+title: myContacts
```
Comment (LizCasey1) [inline Skype/UCWA/myContacts_ref.md:2]: I would make this a tad more descriptive. Maybe "Learn about myContacts, which is a collection of contact resources."

File: Skype/UCWA/onlineMeetingExtension_ref.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+title: onlineMeetingExtension
```
Comment (LizCasey1) [inline Skype/UCWA/onlineMeetingExtension_ref.md:2]: Maybe "Learn about onlineMeetingExtension"

File: Skype/WebSDK/docs/PTLocalUserLocation.md [hunk 4]
```diff
@@ -1,4 +1,7 @@
-
+---
+title: Location
```
Comment (LizCasey1) [inline Skype/WebSDK/docs/PTLocalUserLocation.md:2]: I'd like there to be more than one word in the title, even though these articles are super old/legacy articles. So, maybe you can expand this to a few words?

File: Skype/WebSDK/docs/TermsOfService.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
+---
+title: MICROSOFT SOFTWARE LICENSE TERMS
```
Comment (LizCasey1) [inline Skype/WebSDK/docs/TermsOfService.md:2]: Please take the all-caps off here. I would make this: "Microsoft software license terms for Skype Web SDK."

File: Skype/WebSDK/docs/troubleshooting/auth/AADAuth-DelegatePermissions.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
+---
+title: AAD Auth Failures - Client requesting resource not in requiredResourceAccess list
```
Comment (LizCasey1) [inline Skype/WebSDK/docs/troubleshooting/auth/AADAuth-DelegatePermissions.md:2]: Please remove the dash from this title and everything following the dash. TY!

**Comment by jmirabal80** [comment]
@lindalu-MSFT 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Pull request merged**